terraform/iks-classic/variables.tf

```diff
@@ -18,7 +18,7 @@ variable space {
 
 variable kube_version {
   type        = string
-  default     = "1.19.9"
+  default     = "1.20.6"
   description = "kubectl version"
```

Comment:
Looks like the readme still has 1.19.9. Should it be changed to 1.20.6?

Reply:
good catch. I updated them. thanks. But I still leave the README in root folder untouched.